spec/unit/recipes/default_spec.rb

```diff
@@ -1,50 +0,0 @@
-# This file is auto-generated by the code_generator (one-time action)
```

Comment:
Why are you removing this file? I think it still interesting to see if the default recipe, doing nothing, converge :)
It would validate that the cookbook "compiles" on the test machine.

Reply:
This test doesn't really do anything since we are disabling the pseudo-ohai computation in Chefspec. I don't find it interesting to test an empty recipe in an empty cookbook.